src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1630,16 +1630,6 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
 
                     default:
                         Debug.Assert(local.DeclarationKind == LocalDeclarationKind.UsingVariable);
-                        int slot = GetOrCreateSlot(local);
-                        if (slot >= 0)
-                        {
-                            SetSlotAssigned(slot);
-                            NoteWrite(local, value: null, read: true);
-                        }
-                        else
-                        {
-                            Debug.Assert(_emptyStructTypeCache.IsEmptyStructType(local.Type));
-                        }
                         break;
```

Comment:
I'd prefer to strengthen the assert to say that only pattern and out variables can appear here, but the current implementation of out variables does not provide any way to distinguish them from other locals.